pkg/cli/cmd/validate/cmd.go

```diff
@@ -2,14 +2,15 @@ package validate
 
```

Comment:
I think we should update this command so we can validate and dry run the new istio spec. 

```
go run main.go validate --name istio --flavor istio --version 1.1.7 --print-manifest
```

Reply:
(would be happy to require an application type flag as well, since we have demos, extensions, and meshes)